src/scss/base/_media-config.scss

```diff
@@ -1,16 +1,80 @@
-/* Breakpoints for include-media plugin
-========================================================================== */
-// breakpoints values array - used to generate custom attributes with include-media rwd and for include-media itself
-$breakpoints: (
-    'desktop': size(breakpoint, desktop),
-    'tablet': size(breakpoint, tablet),
-    'mobile': size(breakpoint, mobile)
-);
+/* ==========================================================================
+   Media config
+   ========================================================================== */
 
-// media expressions and how they are aliased in include-media
-$media-expressions: (
-    'screen': 'screen',
-    'landscape': '(orientation: landscape)',
-    'portrait': '(orientation: portrait)',
-    'retina2x': '(-webkit-min-device-pixel-ratio: 2), (min-resolution: 192dpi)',
-);
+// Examples:
+// In mobile-first approach we should use:
+// @include media(mobile) that means (min-width: mobile);
+//
+// To create custom styles (eg. works only above mobile and below tablet) 
+// and avoid override them on another breakpoints, we can use:
+// @include media(mobile, tablet) that means (min-width: mobile) and (max-width: tablet - 1px);
+//
+// To add orientation: portrait or landscape, we can use:
+// @include media(mobile, tablet, portrait) {
+// that means (min-width: mobile) and (max-width: tablet - 1px) and (orientation: portrait);
+
+@mixin media($minWidth, $maxWidth: null, $orientation: null) {
+    @if type-of($minWidth) != number {
+        $minWidth: breakpoint($minWidth);
+    }
+
+    @if ($minWidth != null) {
+        @if ($maxWidth != null) {
+            @if type-of($maxWidth) != number {
+                $maxWidth: breakpoint($maxWidth);
+            }
+
+            @if ($orientation != null) {
+                @if ($orientation == 'portrait' or $orientation == 'landscape') {
```

Comment:
@function font($key) {
    @if map-has-key($fonts, $key) {
        @return map-get($fonts, $key);
    }
}

maybe there should be similar function for orientation?